src/actions/InsuranceActions.js

```diff
@@ -57,7 +57,7 @@ const update = policyDetails => (dispatch, getState) => {
     policyNumberPerson,
     policyNumberFamily,
     discountRate: policyDiscountRate,
-    insuranceProvider: policyProvider,
+    insuranceProviderId: policyProviderId,
```

Comment:
Does this break regular updating of insurance policies through dispensing? 

The regular insurance policy updating passes back the 'form object' from the `FormControl` which uses an insurance provider proper?

Reply:
Good spot. This whole logic needs to be refactored, it's a bad design and an example of too much DRY breaking SRP and SOC all over the place (I can say this because I wrote it!). 

b94a161585a67ab9b62ef93a657703fc1ed34f61.